### PR TITLE
update button styling + stories

### PR DESCRIPTION
**Pull request by smurmus**
### Description:
- updated the `Button` stories to use actual proper color props so that their pages are reflective of each variant
- added an optional `align` prop to `Button` that allows us to define vertical alignment with its parent content
- updated some of the `Banner` styling of its action buttons since there was some weird alignment before....
- updated `SuggestedActions` component to stack buttons on smaller screens + add `align` prop
- updated the `text` variant of a button to have padding rather than none...because what it should really look like is left-alignment + no shifting around on hover state

Screen Shot 2022-07-12 at 7 36 40 PM
Screen Shot 2022-07-14 at 8 02 32 PM
Screen Shot 2022-07-14 at 8 02 49 PM

### CODE REVIEW

File: src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -103,7 +103,11 @@ const Banner = ({
     </Typography>
     <Box
       sx={{
+        mt: { xs: 2, sm: 1, md: 'unset' },
         display: 'flex',
+        aignItems: {
```
Comment (aw2934) [inline src/components/Banner/Banner.tsx:108]: `alignItems`?

File: src/components/Button/Button.stories.tsx [hunk 2]
```diff
@@ -138,6 +130,7 @@ export const textButton = buttonTemplate.bind({});
 
 textButton.args = {
   ...button.args,
+  bg: 'light',
   children: 'Text',
   variant: 'text',
 };
```
Comment (aw2934) [inline src/components/Button/Button.stories.tsx:136]: nit: newline

File: src/components/SuggestedAction/SuggestedAction.tsx [hunk 3]
```diff
@@ -88,12 +88,24 @@ const SuggestedAction = ({
...
           <Typography variant="p3" sx={{ mt: 1 }}>Due on {dueDate}</Typography>
         }
-        <Box sx={{ mt: 2.5, display: 'flex', width: 'fit-content' }}>
-          <Button variant="text" onClick={ctaAction}>
+        <Box
+          sx={{
+            mt: 2.5,
+            display: 'flex',
+            width: 'fit-content',
+            flexWrap: { xs: 'wrap', sm: 'wrap' },
```
Comment (aw2934) [inline src/components/SuggestedAction/SuggestedAction.tsx:96]: do we need to break this down by breakpoint, if they're the same value in each case?

File: src/theme/theme.ts [hunk 4]
```diff
@@ -367,6 +367,7 @@ export const componentOverrides: ThemeOptions['components'] = {
     styleOverrides: {
       text: ({ ownerState, theme }) => (
         {
+          // boxSizing: 'border-box',
```
Comment (aw2934) [inline src/theme/theme.ts:370]: nit: remove comment?

**Review by aw2934 — APPROVED** [review]

File: src/components/Banner/Banner.tsx (hunk 1 above)
Comment (smurmus) [reply src/components/Banner/Banner.tsx:108]: omg 🤦🏽‍♀️

**Review by jlaurelli — APPROVED** [review]

**Pull request merged**